tests/Core/Tokenizer/ExplicitOctalNotationTest.inc

```diff
@@ -0,0 +1,4 @@
+<?php
```

Comment:
Given this is a test of the backfill functionality, the test files would be better named `BackfillExplicitOctalNotationTest.php|inc`